src/assets/ElmAsset.js

```diff
@@ -40,10 +40,10 @@ class ElmAsset extends Asset {
       await this.getConfig(['elm.json'], {load: false});
     }
 
+    options.debug = !this.options.minify;
```

Comment:
Shouldn't this use `!options.production`

Reply:
agree 👍